### ISSUE TITLE
Een toestemmingssysteem is geen requirement voor en uitwisseling.

### ISSUE THREAD

**Issue by MLHoekstra** [minvws/generiekefuncties-toestemming#51]
Ingebracht door Eric

**Comment by VincentvandenBerg** [comment]
Zoals nu beschreven lijkt dit strijdig met de wet i.c.m. de 'keuze' van VWS voor Mitz.
Vermoedelijk wordt bedoeld dat uitwisseling niet alleen o.b.v. uitdrukkelijke toestemming (zoals conform wabvpz) mag plaatsvinden, maar ook bijvoorbeeld o.b.v. veronderstelde toestemming en er in dat geval geen OTV benodigd (vereist) is.

**Comment by stevenvegt** [comment]
> Vermoedelijk wordt bedoeld dat uitwisseling niet alleen o.b.v. uitdrukkelijke toestemming (zoals conform wabvpz) mag plaatsvinden, maar ook bijvoorbeeld o.b.v. veronderstelde toestemming en er in dat geval geen OTV benodigd (vereist) is.

Dat, of dat je als betrokkene geen gebruik wilt maken van Mitz, ook op andere manieren dan een OTV je toestemming voor een uitwisseling moet kunnen geven?

In dat geval heeft hij relatie met #22

**Comment by yellowjacketgreen** [comment]
Klopt. Ik bedoel dit zoals Steven omschrijft; stel dat iemand geen toestemming wil vastleggen in de OTV maar bijvoorbeeld wel op papier bij de arts dan zou de OTV als systeem niet randvoorwaardelijk moeten zijn voor de uitwisseling an sich.
